### DISCUSSION TITLE
Add the possibility to view and filter the notes count

### DISCUSSION THREAD

**Discussion by WawThisGuyIsSoCool** [TumblThreeApp/TumblThree#583] [Ideas]
Firstly i would thank you for this usefull tool

Something i'd like so much to be added is a way to save the note count of a post and reblogged post
and for example being able to add it to the filename template and have the possibility to sort post by note count and filter by it
Do you think it would be possible ?

Thanks again !

**Comment by thomas694** [comment]
This was achieved through the collaboration of all involved.

I think it should be possible to add a new filename template token for it.

**Reply by WawThisGuyIsSoCool** [reply]
I don't understand,  it will be planned to add it ? :)

**Reply by thomas694** [reply]
Yes, this new token will be added.
